### PR TITLE
feat: swap to numeric ids

### PR DESCRIPTION
**Pull request by collectxyznuisance**
update nft to use numeric id: i.e. replace `xyz #123` with `123` as the key and name

### CODE REVIEW

File: contracts/collectxyz-nft-contract/src/execute.rs [hunk 1]
```diff
@@ -293,12 +293,13 @@ pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> StdResult<Response
...
-        tokens().update(deps.storage, &token_id, |old_token| match old_token {
-            Some(old_token) => {
-                let mut new_token = old_token.clone();
-                new_token.image = Some(base64_token_image(&old_token.extension.coordinates));
-                Ok(new_token)
-            }
+        let new_token_id = format!("#{}", token_num);
+        let token = tokens().load(deps.storage, &token_id)?;
+        let mut new_token = token.clone();
+        new_token.name = new_token_id.clone();
```
Comment (collectxyznuisance) [inline contracts/collectxyz-nft-contract/src/execute.rs]: unsure about this, but resolves `value borrowed here after move` on line 301. What is conventional here?

Comment (cocoloco80085) [reply contracts/collectxyz-nft-contract/src/execute.rs]: can just do new_token_id.to_string()

Comment (cocoloco80085) [reply contracts/collectxyz-nft-contract/src/execute.rs]: or just do it in place format!("#{}", token_num).to_string()

File: contracts/collectxyz-nft-contract/src/execute.rs [hunk 2]
```diff
@@ -293,12 +293,13 @@ pub fn migrate(deps: DepsMut, _env: Env, _msg: MigrateMsg) -> StdResult<Response
     let num_tokens = 1 + cw721_contract.token_count(deps.storage)?;
     for token_num in 1..num_tokens {
         let token_id = format!("xyz #{}", token_num);
-        tokens().update(deps.storage, &token_id, |old_token| match old_token {
-            Some(old_token) => {
-                let mut new_token = old_token.clone();
-                new_token.image = Some(base64_token_image(&old_token.extension.coordinates));
-                Ok(new_token)
-            }
+        let new_token_id = format!("#{}", token_num);
```
Comment (0xja) [inline contracts/collectxyz-nft-contract/src/execute.rs]: This format string would give new token ids the format `"#123"`, when we want `"123"`:
```suggestion
        let new_token_id = token_num.to_string();
```

File: contracts/collectxyz-nft-contract/src/execute.rs [hunk 3]
```diff
@@ -73,7 +73,7 @@ pub fn execute_mint(
 
     // create the token
     let num_tokens = 1 + num_tokens;
-    let token_id = format!("xyz #{}", &num_tokens);
+    let token_id = format!("{}", &num_tokens);
```
Comment (0xja) [inline contracts/collectxyz-nft-contract/src/execute.rs]: `num_tokens.to_string()` is probably preferable here.

**Review by 0xja — APPROVED** [review]
lgtm

**Pull request closed without merging**